src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -802,6 +803,14 @@ pCode = try $ do
   result <- manyTill pAny (pCloses open)
   return $ B.codeWith (mkAttr attr) $ T.unwords $ T.lines $ innerText result
 
+pBdo :: PandocMonad m => TagParser m Inlines
+pBdo = try $ do
+  TagOpen _ attr' <- lookAhead $ pSatisfy $ tagOpen (=="bdo") (const True)
+  let attr = toStringAttr attr'
+  contents <- pInTags "bdo" inline
+  let dir = fromMaybe "" $ lookup "dir" attr
+  return $ B.spanWith ("", [], [("dir",dir)]) contents
```

Comment:
nit-pick... without the `"dir"` attribute, this is uselels, so maybe do a `case` on the `Maybe` returned from `lookup` ?

Reply:
Alright. Then I guess when dir is different than rtl, bdo is also useless. Next commit fixes that.

Reply:
well... if the whole doc is rtl, then a ltr span is useful :-)

Reply:
Ah, good point! Ok let me not skip rtl then :)  What about validating that dir must be either rtl or ltr? should that be done in the `pBdo` ?

Reply:
hm.. good question... I guess it wouldn't hurt.. although we're not doing much validation on most attributes...